### PR TITLE
Fix the verbage and align the tool tip left to match other tool tips

### PR DESCRIPTION
**Pull request by nhaugo**
- [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #2532 

### The problem
Verbiage of tool tip should not be boolean logic

### The Solution
Change text

### CODE REVIEW

**Comment by nhaugo** [comment]
@alexpaxton oh noes, i just wanted to adjust the tool tip

**Comment by alexpaxton** [comment]
I pushed a change that will make the tooltips align left instead of inheriting alignment

File: ui/src/style/components/react-tooltips.scss [hunk 1]
```diff
@@ -26,6 +26,7 @@ $tooltip-code-color: $c-potassium;
   word-break: keep-all;
   border-radius: $tooltip-radius;
   text-transform: none;
+  text-align: left;
```
Comment (alexpaxton) [inline ui/src/style/components/react-tooltips.scss]: 💀

File: ui/src/style/components/organizations-table.scss [hunk 2]
```diff
@@ -84,7 +84,6 @@ input[type="text"].form-control.orgs-table--input {
 .orgs-table--public {
   height: 30px;
   margin-right: 4px;
-  text-align: center;
```
Comment (alexpaxton) [inline ui/src/style/components/organizations-table.scss]: 💀

**Pull request closed without merging**